tests/xtro-sharpie/api-annotations-dotnet/iOS-AVFoundation.ignore

```diff
@@ -12,3 +12,7 @@
 
 ### deprecated in iOS8
 !missing-protocol-member! AVAudioPlayerDelegate::audioPlayerEndInterruption:withFlags: not found
+
+## https://github.com/xamarin/xamarin-macios/issues/16229 prevents crashing instead of return null
```

Comment:
I think I would add that these selectors have been bound manually:

```suggestion
## https://github.com/xamarin/xamarin-macios/issues/16229 prevents crashing instead of return null
## so these selectors have been bound manually
```

and likewise for all the other comments below